flaskr/api.py

```diff
@@ -16,48 +19,31 @@ def __init__(self):
 
     def get(self):
         # get all files in the ./static/uploads directory
-        files = [f for f in os.listdir(UPLOAD_DIR)]
+        files = [f for f in os.listdir(UPLOAD_DIR + '/dicoms')]
         return {
-                'dir' : UPLOAD_DIR, 
+                'pathRelative': DICOM_PATH_RELATIVE,
+                'pathAbsolute': DICOM_PATH_ABSOLUTE,
                 'count' : len(files),
                 'files' : files,
```

Comment:
Could anyone perform this GET query and receive these results? Or are there some restrictions?

Reply:
This api was only tested locally, so no in that sense. There are, however, no authentication mechanisms. We will implement them before we make the app publicly available. I have no experience with this sort of thing, so for more detail it's probably a good idea to ask @Qwebeck

Reply:
I am not sure if we do need to implement an autorization because it can be time consuming. Also, as far as we don't store any private date related to user (e.g. name, surname, phone number, etc) we don't need to secure it. So I think, that at this time we actually don't need to implement it. 

Nevertheless, I do think, that we should implemented an authentication, in order to allow user to save an application state and also in order to be able to differentiate between processes.

I think we can use an ip address that comes with every request ([here](https://stackoverflow.com/questions/3759981/get-ip-address-of-visitors-using-flask-for-python) is an example of how to get it) in order to achieve this.